docs/how-to-guides/web-portal/web-portal-24-04-or-later/use-security-profiles.md

```diff
@@ -0,0 +1,74 @@
+(how-to-web-portal-use-security-profiles)=
+# How to use security profiles
+
+You can use security profiles to automatically audit and fix your managed instances on a schedule using the [Ubuntu Security Guide (USG)](https://documentation.ubuntu.com/security/docs/compliance/usg/).
+
+```{note}
+This feature is only available in self-hosted **Landscape 25.04** and later.
+```
+
+## Create a security profile
+
+On each instance you want to manage, follow the steps in [Install Ubuntu Security Guide](https://documentation.ubuntu.com/security/docs/compliance/usg/install-usg/), then:
+
+1. Edit the client configuration file `/etc/landscape/client.conf`, add this line:
+
+    ```
+    include_manager_plugins = UsgManager
+    ```
+
+    If your `client.conf` already includes an `include_manager_plugins` line, then add `UsgManager` to it. For example:
+
+    ```
+    include_manager_plugins = ScriptExecution,UsgManager
+    ```
+
+2. Restart Landscape Client:
+
+    ```
+    sudo systemctl restart landscape-client
+    ```
+
+From the web portal:
+
+1. Click **Profiles**
+2. Click **Security Profiles**
+3. Click **Add security profile**
+
+In the security profile creation form, complete the following fields:
+
+- **Profile name**: name of the profile
```

Comment:
```suggestion
- **Profile name**: Name of the profile
```
Is there a logical example that would help someone understand how these are used? If yes, you could add "For example, XYZ."

If there's not a good example (e.g., if the example would be a date, or something unclear), then no need to add